controllers/topics.controller.js

```diff
@@ -8,4 +8,14 @@ exports.getTopics = async (req, res, next) => {
   } catch (err) {
     next(err);
   }
-};
\ No newline at end of file
+};
+
+exports.getEndpoints = async (req, res, next) => {
+  try {
+    const endPoints = await fetchEndpoints();
+    res.status(200).send({ endPoints });
+  } catch (err) {
+    next(err);
```

Comment:
What error are you expecting from this end point? Is this catch really necessary?